analysers/analyser_merge_bicycle_parking_FR_paris.py

```diff
@@ -122,8 +122,9 @@ def __init__(self, config, logger = None):
             Mapping(
                 select = Select(
                     types = ["nodes"],
-                    tags = {"amenity": "motorcycle_parking"}),
-                conflationDistance = 50,
+                    tags = [{"amenity": "motorcycle_parking"},
+                            {"amenity": "bicycle_parking"}])
```

Comment:
`tags = {"amenity": ["motorcycle_parking", "bicycle_parking"]}`
Use this, it is more efficient.

Reply:
Oh, I missed that this syntax was actually valid. This should be fixed :)